lib/board.rb

```diff
@@ -1,8 +1,16 @@
-#!usr/bin/ruby
 # frozen_string_literal: true
 
+require_relative 'UserInterface'
+
 class Board
-  def display_board
+  include UserInterface
+  attr_accessor :board
+
+  def initialize
+    @board = Array.new(9)
+  end
+
+  def display_init_board
     row = [
```

Comment:
Just few change, let move all our `puts` inside main.rb.
According to the instructions on pathwright, our `puts` and `gets.chomp` are to be inside the `main.rb`